schemas/clab.schema.json

```diff
@@ -238,7 +238,7 @@
                     "minItems": 2,
                     "items": {
                         "type": "string",
-                        "pattern": "^[\\w\\s-/]+:[\\w\\s-/]+$"
```

Comment:
let's maybe use the following 

```
^\\S+:\\S+$
```
which allows any non whitespace char for node and link portions

your current example will not highlight a missing node name, which is not great

Reply:
Yeah, that looks better than what I tried. I didn't think to check not having a node name